### PR TITLE
CXP-1375: other method to add link between type:string and format property

### PR DESCRIPTION
**Pull request by hogren**

### CODE REVIEW

File: content/mapping/product/0.0.4/schema [hunk 1]
```diff
@@ -40,6 +40,8 @@
       "type": "string"
     },
     "propertyWithSimpleType": {
+    },
```
Comment (masacc) [inline content/mapping/product/0.0.4/schema:43]: useless

**Comment by masacc** [comment]
Nice solution, we can apply it in next version of the schema. GG @hogren

**Comment by masacc** [comment]
@hogren you can close this PR but we will use this solution for next version (0.0.5). The 0.0.4 version is already merged on master-octopus

**Review by tseho — CHANGES_REQUESTED** [review]
the documentation should contains only a copy of the schema from pim-community-dev, they should be identical, the changes should be in CE first in newer schema versions.

**Pull request closed without merging**